pulumi/python/kubernetes/observability/otel-operator/opentelemetry-operator.yaml

```diff
@@ -61,6 +61,131 @@ spec:
             description: InstrumentationSpec defines the desired state of OpenTelemetry
               SDK and instrumentation.
             properties:
+              dotnet:
+                description: DotNet defines configuration for DotNet auto-instrumentation.
```

Comment:
What is DotNet in this context?  Should the description here link to some project page so the reader can find the docs and understand what is being configured here?

Reply:
I will put in a link to the OTEL operator repo in the source to folks can track back to it if they need to.